src/styles/hand.scss

```diff
@@ -0,0 +1,43 @@
+.hand {
+  position: absolute;
+  top: 0;
+  left: 49%;
+
+  transform-origin: center bottom;
+  transform: translate(-50%, -50%);
+
+  &--minute {
+    top: 20%;
+    width: 2 * $hand-size;
+    height: $hand-height;
+    background-color: $hand-minute-color;
+    animation: rotateMinuteHand 60s infinite linear;
+    border-radius: 10px;
+  }
+
+  &--second {
+    width: $hand-size;
+    height: $hand-height + 20%;
+    background-color: $hand-second-color;
+    animation: rotateSecondHand 5s infinite linear;
+    border-radius: 5px;
+  }
+}
+
+@keyframes rotateMinuteHand {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotateSecondHand {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
These 2 animations do the same thing, so you need to remove one

Reply:
not fixed

Reply:
this animations do the same thing but if I will remove one, one of them will not perform animation

Reply:
the first makes a full revolution in 60 seconds, and the second makes a revolution in 5 seconds

Reply:
@vneholiuk this isn't true, you don't set timing in this animations, u can just leave one of them and call it `rotateHand` and then in `&--second` you make `animation: rotateHand 5s infinite linear;` and in `&--minute` you do `animation: rotateHand 60s infinite linear;`